### ISSUE TITLE
PySlices[Shell]

### ISSUE THREAD

**Issue by RandyERaymond** [wxWidgets/Phoenix#799]
**Operating system**: WIndows 10
**wxPython version**: 4.01
**Stock or custom build**: Stock [download]
**Python version**: 3.6.4
**Stock or custom build**: Stock [download]

**Description of the problem**:  Whenever I run PySlices or PySlicesShell from the install folder, I always get an error with the history. The error first occurs during program load:

![captureerror](https://user-images.githubusercontent.com/11489809/37978538-e67a3e4e-31ab-11e8-8a62-8e3f47419ab1.JPG)
![captureerror2](https://user-images.githubusercontent.com/11489809/37978744-6475d8c6-31ac-11e8-9ee9-12ed5db61570.JPG)

And again if I try to do anything with Options->History. I would like to use PySlices, but it seems that history is an important function.

```python
The code is exactly what is distributed with the wxPython package to Windows users. 
```

**Comment by RobinD42** [comment]
Closed by #802

**Comment by RandyERaymond** [comment]
Thank you
